lib/apivore/rspec_matchers.rb

```diff
@@ -22,8 +22,18 @@ module RspecMatchers
       end
 
       define_method :fetch_master_swagger do
-        req = Net::HTTP.get(master_swagger_host, "/swagger.json")
-        JSON.parse(req)
+        res =
+          if master_swagger_url.starts_with? 'http'
```

Comment:
`Net::HTTP.get` takes either a host string and path parameters or a full URI. One distinguishing feature of a full URI is that the protocol is specified. This logic is simple and adequate to allow http or https.